linux-admin/part-2-tasks/tasks_othman.md

```diff
@@ -0,0 +1,154 @@
+# First step on command line.
+1. Display your current directory.
+
+pwd
+
+2. Change to the /etc directory.
+
+cd /etc
+
+3. Now change to your home directory using only three key presses.
+
+cd
+
+4. Go to the parent directory of the current directory.
+
+cd ..
+
+5. Go to the root directory.
+
+cd /
+
+6. List a long listing of the root directory using two ways.
+
+ll 
+ls -l
+
+7. Using man pages list all files under /usr/bin and show list of files in long listing format,human readable size format,and show the oldest files first.
+
+ls -lhtr
+
+8. Stay where you are, and list the contents of /bin and /sbin.
+
+ll /bin
+ll /sbin
+
+9. List the files in /boot in a human readable format.
+
+ll /boot -h
+
+10. Create in one command the directories ~/dir1/dir2/dir3 (dir3 is a subdirectory
+from dir2, and dir2 is a subdirectory from dir1 ).
+
+mkdir ~/dir1/dir2/dir3 -p
+
+11. create /tmp/test-fld folder and /tmp/test-File file and move /tmp/test-File inside /tmp/test-fld. Now remove /tmp/test-fld using two ways.
+
+mkdir /tmp/test-fld 
+touch /tmp/test-File
+mv /tmp/test-File /tmp/test-fld
+
+rmdir /tmp/test-fld --ignore-fail-on-non-empty
```

Comment:
@othmanemad both commands for deletion the directory will not work as the directory is not empty so you have to use one of the below commands:
```bash
1) rm -r /tmp/test-fld (which will delete the directory alongside its content)
2) rm /tmp/test-File; rmdir  /tmp/test-fld (delete the file first then delete the dir)
```

Reply:
The rmdir deletes any folder even if it has files in it

Reply:
no `rm -r` deletes any folder with it's content and you can test it out